src/helper/findTabById.tsx

```diff
@@ -0,0 +1,7 @@
+import { Tab } from '../types/Tab';
+
+export const findTabById = (tabs: Tab[], tabId: string) => {
+  const selectedTab = tabs.find(tab => tab.id === tabId);
```

Comment:
Consider renaming `selectedTab` to `foundTab` or `matchingTab` for better readability and clarity. 🤔